internal/deploy/workspace.go

```diff
@@ -119,6 +110,25 @@ func ResolveLocalWorkspacePath(ws string, local bool) (string, error) {
 	return ws, nil
 }
 
+// resolveLocalRepo tries to find the repository with the given name in the GOPATH
+// the repository must have its full name starting from $GOPATH/src
+func resolveLocalRepo(repo string) (string, error) {
```

Comment:
I think there is a problem here. GOPATH should point to the workspace dir. In this directory there are "src" and "bin" subdirectories, and project sources should be located under "src" [(docs)](https://go.dev/doc/gopath_code#GOPATH)
On my system the "src" dir is not taken into account and I go this:
```
% echo $GOPATH
/Users/tsm/Documents/development/golang
% ./bin/kyma-darwin alpha deploy --source=local
- Kustomize ready
Error: error resolving lifecycle-manager: Local Kyma source directory '/Users/tsm/Documents/development/golang/github.com/kyma-project/lifecycle-manager' not found
% file $GOPATH/src/github.com/kyma-project/lifecycle-manager
/Users/tsm/Documents/development/golang/src/github.com/kyma-project/lifecycle-manager: directory
```

Reply:
Thanks for catching that! Yes indeed there is a bug there; the `src` path element is missing.
Fixed it.